tests/playwright/src/model/ai-lab-page.ts

```diff
@@ -0,0 +1,130 @@
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Locator, Page } from '@playwright/test';
+import { expect as playExpect } from '@playwright/test';
+import { handleConfirmationDialog } from '@podman-desktop/tests-playwright';
+
+export class AILabPage {
+  readonly page: Page;
+  readonly webview: Page;
+  readonly heading: Locator;
+  readonly navigationBar: Locator;
+
+  readonly recipesCatalogButton: Locator;
+  readonly runningAppsButton: Locator;
+  readonly catalogButton: Locator;
+  readonly servicesButton: Locator;
+  readonly playgroundsButton: Locator;
+
+  readonly recipesCatalogPage: Locator;
+  readonly recipesCatalogContent: Locator;
+  readonly recipesCatalogNaturalLanguageProcessing: Locator;
+  readonly recipesCatalogAudio: Locator;
+  readonly recipesCatalogComputerVision: Locator;
+
+  readonly startRecipeButton: Locator;
+  readonly confirmStartRecipeButton: Locator;
+  readonly recipeStatus: Locator;
+
+  readonly applicationDetailsPanel: Locator;
+  readonly openAIAppButton: Locator;
+  readonly deleteAIAppButton: Locator;
+
+  constructor(page: Page, webview: Page) {
+    this.page = page;
+    this.webview = webview;
+    this.heading = webview.getByLabel('Welcome to Podman AI Lab');
+    this.navigationBar = webview.getByLabel('PreferencesNavigation');
+
+    this.recipesCatalogButton = this.navigationBar.getByLabel('Recipes Catalog', { exact: true });
+    this.runningAppsButton = this.navigationBar.getByLabel('Running');
+    this.catalogButton = this.navigationBar.getByLabel('Catalog', { exact: true });
+    this.servicesButton = this.navigationBar.getByLabel('Services');
+    this.playgroundsButton = this.navigationBar.getByLabel('Playgrounds');
+
+    this.recipesCatalogPage = webview.getByRole('region', { name: 'Recipe Catalog' })
+    this.recipesCatalogContent = this.recipesCatalogPage.getByRole('region', { name: 'content', exact: true }).first();
+    this.recipesCatalogNaturalLanguageProcessing = this.recipesCatalogContent.getByRole('region', { name: 'Natural Language Processing', exact: true });
+    this.recipesCatalogAudio = this.recipesCatalogContent.getByRole('region', { name: 'Audio', exact: true });
+    this.recipesCatalogComputerVision = this.recipesCatalogContent.getByRole('region', { name: 'Computer Vision', exact: true });
+
+    this.startRecipeButton = this.webview.getByRole('button', { name: 'Start recipe' });
+    this.confirmStartRecipeButton = this.webview.getByRole('button', { name: /Start(\s+([A-Za-z]+\s+)+)recipe/i });
```

Comment:
I'm confused of which buttons these are, or more specifically as far as I can see the start receipy button is always named "Start AI App", why are we not locating by that name, since we wouldnt't need to use regex and to be honest I can't even identiy which button "Start recipe" refers to.

Reply:
that's a different button that appears in the Recipe dialog, after "Start" is clicked

Reply:
there are two buttons when you open AI App Details Page (Recipes Catalog -> App Details)
there is a `Start` button in top right corner that has `aria-label` `Start recipe`
and a second button after you click on `Start recipe` which is `Start <appName> recipe`